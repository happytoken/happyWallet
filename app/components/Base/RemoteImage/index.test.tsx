import React from 'react';
import { shallow } from 'enzyme';
import RemoteImage from './';

describe('RemoteImage', () => {
	it('should render correctly', () => {
		const wrapper = shallow(
			<RemoteImage
				source={{
					uri: `https://hld-chain.oss-cn-beijing.aliyuncs.com/tokens/dai.svg`,
				}}
			/>
		);
		expect(wrapper).toMatchSnapshot();
	});
});
