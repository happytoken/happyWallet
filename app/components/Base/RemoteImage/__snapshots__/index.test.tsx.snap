// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RemoteImage should render correctly 1`] = `
<ComponentErrorBoundary
  componentLabel="RemoteImage-SVG"
>
  <View
    style={Object {}}
  >
    <SvgCssUri
      fill="black"
      height="100%"
      source={
        Object {
          "uri": "https://hld-chain.oss-cn-beijing.aliyuncs.com/tokens/dai.svg",
        }
      }
      uri="https://hld-chain.oss-cn-beijing.aliyuncs.com/tokens/dai.svg"
      width="100%"
    />
  </View>
</ComponentErrorBoundary>
`;
